app/scripts/modules/ecs/src/serverGroup/configure/wizard/CloneServerGroup.ecs.controller.js

```diff
@@ -241,5 +241,21 @@ module(ECS_SERVERGROUP_CONFIGURE_WIZARD_CLONESERVERGROUP_ECS_CONTROLLER, [
     $scope.configureCommand = function (query) {
       return ecsServerGroupConfigurationService.configureCommand($scope.command, query);
     };
+
+    // TODO: Migrate horizontalScaling component to react and move this logic there
+    $scope.capacityProviderState = {
+      useCapacityProviders:
+        $scope.command.capacityProviderStrategies && $scope.command.capacityProviderStrategies.length > 0
```

Comment:
Looks great to me just a minor suggestion.
I guess the condition itself will return true or false. So we don't need to return true or false specifically.

Reply:
you're right! i will make that change